src/synapse/authProvider.js

```diff
@@ -1,5 +1,22 @@
 import { fetchUtils } from "react-admin";
 
+const ensureHttpsForUrl = (url) => {
+  if (/^https:\/\//i.test(url)) {
```

Comment:
This `if` is probably not required, since the `replace` will simply replace https:// by https://

Reply:
I thought of return early & often here - in a [regex tester](https://www.regextester.com/93652) `/http?:\/\//g` only matches `http://` but not `https://` so we only replace if necessary

Reply:
Right, so either remove the `?` after `http` in line 7 or put a `.` in front of it. Right now it makes the `p` optional which is not what we want.